go/vt/vtctl/grpcvtctldserver/server_test.go

```diff
@@ -5865,6 +5865,71 @@ func TestGetTablets(t *testing.T) {
 	}
 }
 
+func TestGetTopology(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+	ts := memorytopo.NewServer("cell1", "cell2", "cell3")
+	vtctld := testutil.NewVtctldServerWithTabletManagerClient(t, ts, nil, func(ts *topo.Server) vtctlservicepb.VtctldServer {
+		return NewVtctldServer(ts)
+	})
+
+	err := ts.CreateKeyspace(ctx, "keyspace1", &topodatapb.Keyspace{})
+	require.NoError(t, err)
+
+	err = ts.CreateTablet(ctx, &topodatapb.Tablet{
+		Alias:         &topodatapb.TabletAlias{Cell: "cell1", Uid: 100},
+		Hostname:      "localhost",
+		Keyspace:      "keyspace1",
+		MysqlHostname: "localhost",
+		MysqlPort:     17100,
+	})
+	require.NoError(t, err)
+
+	resp, err := vtctld.GetTopology(ctx, &vtctldatapb.GetTopologyRequest{})
+	assert.NoError(t, err)
+	assert.ElementsMatch(t, []*vtctldatapb.TopologyCell{
```

Comment:
when comparing protobuf message types, you should use `utils.MustMatch` (like the newer tests do)